physionet-django/user/test_views.py

```diff
@@ -266,6 +267,34 @@ def test_edit_emails(self):
         request = self.client.get(reverse('verify_email', args=(uidb64, token)))
         self.assertTrue(AssociatedEmail.objects.get(email='tester0@mit.edu').is_verified)
 
+    def test_purgeaccounts(self):
+        # Test 0: login
+        self.client.login(username='admin@mit.edu', password='Tester11!')
+        self.assertEqual(int(self.client.session['_auth_user_id']), self.user.pk)
+
+        # Test 1: add email
+        self.client.post(reverse('edit_emails'), data={
+            'add_email': [''], 'email': 'tester2@mit.edu'})
+        self.assertIsNotNone(AssociatedEmail.objects.filter(email='tester2@mit.edu'))
+
+        # Test 2: add email to be removed
+        self.client.post(reverse('edit_emails'), data={
+            'add_email': [''], 'email': 'tester1@mit.edu'})
+        self.assertIsNotNone(AssociatedEmail.objects.filter(email='tester1@mit.edu'))
+
+        email1 = AssociatedEmail.objects.get(email='tester1@mit.edu')
+        email1.added_date -= timezone.timedelta(days=30)
```

Comment:
is this intentional?

Reply:
Either that or `email1.added_date = timezone.timedelta(days-=30)`  I had to set the added date 30 days into the past so that it would be removed.

Reply:
got it, thanks